task/02-numbers-tasks.js

```diff
@@ -54,7 +54,14 @@ function getCicleCircumference(radius) {
  *  -3, 3  => 0
  */
 function getAverage(value1, value2) {
-    throw new Error('Not implemented');
+    const sum = value1 + value2;
+    if(sum > Number.MAX_VALUE){
+        const temp1 = value1/Number.MAX_VALUE;
+        const temp2 = value2/Number.MAX_VALUE;
+        const average = (temp1 + temp2)/2;
+        return average * Number.MAX_VALUE;
```

Comment:
Very complex calculation. You can use `value1 / 2 + value2 / 2` expression and it will work as your code :)